app/src/main/java/org/oppia/android/app/devoptions/vieweventlogs/ViewEventLogsViewModel.kt

```diff
@@ -14,11 +15,17 @@ import javax.inject.Inject
 @FragmentScope
 class ViewEventLogsViewModel @Inject constructor(
   debugAnalyticsEventLogger: DebugAnalyticsEventLogger,
+  debugFirestoreEventLogger: DebugFirestoreEventLogger,
   private val machineLocale: OppiaLocale.MachineLocale,
   private val resourceHandler: AppLanguageResourceHandler
 ) : ObservableViewModel() {
+  // Retrieves events from cache that are meant to be uploaded to Firebase Firestore.
+  private val firestoreEvents = debugFirestoreEventLogger.getEventList()
 
-  private val eventList = debugAnalyticsEventLogger.getEventList()
+  // Retrieves events from cache that are meant to be uploaded to Firebase Analytics.
+  private val analyticsEvents = debugAnalyticsEventLogger.getEventList()
+
+  private val eventList = analyticsEvents + firestoreEvents
```

Comment:
Should these maybe be sorted by timestamp so that the two sets of events aren't concatenated?

Reply:
I concatenated these because they are 2 seperate lists from different loggers. But I have made a change such that I add the lists to `eventList` then sort by timestamp.